src/components/ExamplesFromPkg.test.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Please disable only the specific line, not the whole file. You can use `eslint-disable-next-line` for this :)

Reply:
Well I'm already learning a lot! I thought I was only disabling eslint on my own editor as it came up on the GUI, I didn't even realise it was impacting the file as a whole.

Reply:
This repository has more or less all the bells and whistles in place regarding that: There are settings in place to ensure your editor will use the linting rules set by this repository, and if one of them is violated, a check on the PR will fail.
One thing that helped me a lot early on (and I'm still doing it today) is to look through all the changes I am actually committing when doing a commit. Some editors have visual tools build in that can help with this. but there is also a free tool called SourceTree that is used by some.